app/packages/operators/src/state.ts

```diff
@@ -579,11 +584,18 @@ export const useOperatorPrompt = () => {
     },
     [operator, promptingOperator, cachedResolvedInput]
   );
+
+  const setOperatorPromptCloseState = useSetRecoilState(
+    operatorPromptCloseState
+  );
+
   const close = () => {
```

Comment:
What exactly does this `close` state entail? Traditionally, an `on_close` hook would mean clicking the 'x' button. 

Does it mean when the prompt has been successfully executed? Does it include the case when prompt modal is canceled and closed? Can we use a more descriptive name?